Source/WebCore/platform/graphics/BifurcatedGraphicsContext.cpp

```diff
@@ -202,6 +202,14 @@ void BifurcatedGraphicsContext::fillRect(const FloatRect& rect, Gradient& gradie
     VERIFY_STATE_SYNCHRONIZATION();
 }
 
+void BifurcatedGraphicsContext::fillRect(const FloatRect& rect, Gradient& gradient, const AffineTransform& gradientSpaceTransform)
```

Comment:
Why does the change cover the `fillRect()` only? Do not we need to add a similar variant to `strokeRect()`?

Reply:
No, stroking is not needed, at least not for the purposes of LBSE gradients. I need to be able to fill a rectangle using the stroke, gradient in a clean way.